src/Nethermind/Nethermind.TxPool/TxHandlingOptions.cs

```diff
@@ -42,12 +42,7 @@ public enum TxHandlingOptions
         /// Allows transaction to be signed by node even if its already signed
         /// </summary>
         AllowReplacingSignature = 8,
-        
-        /// <summary>
-        /// Indicates transaction is re-added after reorganisation
-        /// </summary>
-        Reorganisation = 16 | PersistentBroadcast,
```

Comment:
So the idea here is that both OE and Geth do prioritize somewhat txs that were reorged.

Reply:
yes, but it was not used -> probably you wanted to do it other way round PersistentBroadcast -> 8 | Reorganization.

Reply:
No it was fine like it was, but after recent refactorings it became obsolete